datajunction-server/datajunction_server/internal/namespaces.py

```diff
@@ -33,7 +37,10 @@ def get_nodes_in_namespace(
         NodeRevision.mode,
         NodeRevision.updated_at,
     ).where(
-        col(Node.namespace).contains(namespace),  # pylint: disable=no-member
+        or_(
+            col(Node.namespace).like(f"{namespace}.%"),  # pylint: disable=no-member
```

Comment:
Are you sure we need the dot between `{namespace}` and `%` ? This could cause some confusion imo.

Reply:
@agorajek I think the problem here is that if we don't have this `.`, we'll end up matching more than we should. For example, let's say we want to search for `namespace=foo`, without `.` it'll match to `foobar.group1` and `foo.bar.group1`, but `foobar.group1` shouldn't be included in there because it doesn't actually match the full namespace chunk.

Reply:
I see your point. I think I used the namespace prefix somewhere in the code that was using the logic w/o an implicit dot, but that might have been for listing only. On the 2nd thought I think your approach is safer here, because this is deleting.